spring-integration-core/src/test/java/org/springframework/integration/dsl/manualflow/ManualFlowTests.java

```diff
@@ -494,7 +495,7 @@ public Date foo() {
 
 		@Override
 		protected IntegrationFlowDefinition<?> buildFlow() {
-			return from(() -> new GenericMessage<>("flowAdapterMessage"),
+			return from((MessageSource<?>) () -> new GenericMessage<>("flowAdapterMessage"),
```

Comment:
I wonder if that would better to rename to `fromSupplier()`. And yeah... remove that `from(Supplier)` altogether.

Breaking change, agreed, but no choice.

I will find this explicit casting style more annoying than removed method.

WDYT?